### PR TITLE
Update js/jquery.custom-radio-checkbox.js

### PR DESCRIPTION
**Pull request by cyriltata**
anywhere you see //#CHANGE HERE is my alteration to the previous latest version.
I noticed that though the state of the 'fakeElement' was changing, the actual HTML element was not changing. So I enabled this and also triggered 2 events ('check' and 'uncheck') depending on the status of the element.
With these new events, one can then do the following implementation on the front-end (usage)to get what actually happened to the HTML element

<code>
$('checkbox').bind("check", function(e) {
    //box is checked
    //use $(this) to do anything with box like $(this).val() to get value
}).bind("uncheck", function(e) {
    //box is unchecked
    //use $(this) to do anything with box like $(this).val() to get value
});
</code>

### CODE REVIEW

File: js/jquery.custom-radio-checkbox.js [hunk 1]
```diff
@@ -22,7 +22,7 @@
         var $this = $(this);
         // only trigger if the input is not inside a label
         if (!$this.closest('label')[0]) {
-          $this.prev().trigger('change.crc', [true]);
+          $this.prev().trigger('change.crc', true);
```
Comment (dciccale) [inline js/jquery.custom-radio-checkbox.js:25]: don't do this, as the arguments passed through trigger must be an array, see .trigger() docs http://api.jquery.com/trigger/

File: js/jquery.custom-radio-checkbox.js [hunk 2]
```diff
@@ -38,7 +38,7 @@
           input = inputs[l];
 
           // fake input
-          fakeInputElem = $('<i>').addClass(type + (input.checked ? ' ' + type + checkedSuffix : '')).bind('click.crc', forceChange);
+          fakeInputElem = $('<i>').addClass(type + (input.checked ? ' ' + type + checkedSuffix : '')).bind('click', forceChange); //#CHANGE HERE (click.crc to click)
```
Comment (dciccale) [inline js/jquery.custom-radio-checkbox.js:41]: is a good practice to keep all events related to the plugin with a namespace, in this case <code>crc</code>, please keep it

File: js/jquery.custom-radio-checkbox.js [hunk 3]
```diff
@@ -109,9 +116,19 @@
         // bind checkbox change event
         chs.bind('change.crc', function (e, force) {
           // force change state
-          if (force) {
+		  //#CHANGE HERE
+		  if (force) {
+			  if($(this).attr('checked') === "checked"){
```
Comment (dciccale) [inline js/jquery.custom-radio-checkbox.js:121]: you are not caching $(this)! always do it if you are going to use it more than once.

also see that i was using native javascript to check or uncheck inputs, it is much much faster than using jQuery's attr method.

**Comment by dciccale** [comment]
i appreciate your pull request but you already can do what you are asking for.

just do this:

``` javascript
$(function () {
  $('input[type="checkbox"]').change(function () {
    alert(this.checked);
  });
});
```

using native this.checked you already know if the checkbox is checked or not.

you can do the same thing for radios.

thanks for your participation and hope you don't take it bad.

File: js/jquery.custom-radio-checkbox.js (hunk 2 above)
Comment (cyriltata) [inline js/jquery.custom-radio-checkbox.js:41]: and how do you detect when the event is fired? b

File: js/jquery.custom-radio-checkbox.js (hunk 3 above)
Comment (cyriltata) [inline js/jquery.custom-radio-checkbox.js:121]: yes true, but then the actual html elements were not being checked or unchecked<in short>

File: js/jquery.custom-radio-checkbox.js (hunk 1 above)
Comment (cyriltata) [inline js/jquery.custom-radio-checkbox.js:25]: !!this was just a typo at the end -  actually what i have is when brackets

**Pull request closed without merging**